platforms/nodejs/versionsToBuild.txt

```diff
@@ -16,6 +16,7 @@
 14.17.4
 14.17.6
 14.18.3
+14.19.1
```

Comment:
For updating node versions, are we missing to update the following files ? -
1. build/__nodeVersions.sh
2. src/BuildScriptGenerator/Node/NodeVersions.cs